helm-chart-sources/edge/templates/daemonset.yaml

```diff
@@ -162,6 +162,9 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
+      {{- if .Values.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
+      {{- end }
```

Comment:
s/}/}}/

Reply:
Hmm, less obscurely: this should have a double close brace, not just a single.

Reply:
good catch!  fixed